js/index.js

```diff
@@ -26,17 +34,64 @@ function removeProduct(event) {
   const target = event.currentTarget;
   console.log('The target in remove is:', target);
   //... your code goes here
+  const productToBeDeleted = target.parentElement.parentElement;
+  const allProducts = document.querySelector('tbody');
+
+  allProducts.removeChild(productToBeDeleted);
 }
 
```

Comment:
Aqui na removeProduct() não está aparecendo o início da função pra mim (a primeira linha de declaração da função), talvez seja só um buguizinho de visualização aqui... 
e no fina, faltou apenas "chamar" novamente a calculateAll(), pra recalcular o total, após a remoção de algum produto.